package.json

```diff
@@ -47,6 +47,7 @@
     "@types/enzyme-adapter-react-16": "^1.0.3",
     "@types/jest": "^24.0.18",
     "@types/react-redux": "^6.0.10",
+    "@types/styled-components": "^4.1.18",
```

Comment:
Please remove this dependency. It is not being used in this project, and we don't plan on using it.